edx_sga/sga.py

```diff
@@ -55,6 +56,7 @@ class StaffGradedAssignmentXBlock(XBlock):
     """
     has_score = True
     icon_class = 'problem'
+    STUDENT_FILEUPLOAD_MAX_SIZE = 4 * 1000 * 1000  # 4 MB
```

Comment:
plus if it's already defined, then you should be using that variable instead of redefining it here.